### PR TITLE
Search results are now grouped. Groups are currently set inside the i…

### PR DESCRIPTION
**Pull request by blattmann**
Search results are now grouped. 
Groups are currently set inside the index.html and should be replaced via backend.

### CODE REVIEW

File: contribs/gmf/less/map.less [hunk 1]
```diff
@@ -12,9 +12,16 @@
     background-color: @map-tools-bg-color;
     border: solid 1px @map-tools-color;
     color: @map-tools-color;
-    &:hover,
+    &:hover {
+      background-color: @onhover-color;
+    }
     &:focus {
       background-color: @onhover-color;
+      background-color: @map-tools-bg-color;
```
Comment (fgravin) [inline contribs/gmf/less/map.less]: if you put no backgound-color ?

File: contribs/gmf/less/map.less [hunk 2]
```diff
@@ -22,6 +29,20 @@
   }
 }
 
+/**
+ * Only affects webkit desktop browsers
+ */
+@media screen and (-webkit-min-device-pixel-ratio:0) {
```
Comment (fgravin) [inline contribs/gmf/less/map.less]: what is it ? is it needed ?

File: contribs/gmf/less/mobile.less [hunk 3]
```diff
@@ -64,7 +64,7 @@ span.twitter-typeahead {
   background-color: white;
   border: solid 1px black;
   .search-header {
-    display: none;
+    display: block;
```
Comment (fgravin) [inline contribs/gmf/less/mobile.less]: i guess you can remove it, it was probably already block no ?

File: contribs/gmf/src/controllers/abstract.js [hunk 4]
```diff
@@ -41,7 +42,9 @@ goog.require('ngeo.StateManager');
 gmf.AbstractController = function(
     config, defaultLang, langUrls, gettextCatalog, ngeoGetBrowserLanguage,
     $scope, ngeoStateManager, ngeoFeatureOverlayMgr,
-    gmfThemes, fulltextsearchUrl) {
+    gmfThemes, fulltextsearchUrl, gmfSearchGroups) {
+
+      console.log('gmfSearchGroups: ', gmfSearchGroups);
```
Comment (fgravin) [inline contribs/gmf/src/controllers/abstract.js]: remove the log

**Comment by fgravin** [comment]
Some minor comments but it is good.

BTW you mixed 2 PR again, the zoom button one and the group values, but we will deal with it.
Please check my comments, i'll merge then.

File: contribs/gmf/less/mobile.less (hunk 3 above)
Comment (blattmann) [inline contribs/gmf/less/mobile.less]: nope. it was still "none" in the latest master, so i changed it to "block".

File: contribs/gmf/src/controllers/abstract.js (hunk 4 above)
Comment (blattmann) [inline contribs/gmf/src/controllers/abstract.js]: done.

File: contribs/gmf/less/map.less (hunk 2 above)
Comment (blattmann) [inline contribs/gmf/less/map.less]: yes, it adds the light blue shadow around the buttons. it is the default behaviour of webkit browsers. i guess we want to keep it?

File: contribs/gmf/less/map.less (hunk 1 above)
Comment (blattmann) [inline contribs/gmf/less/map.less]: no background-color = transparent

**Comment by pgiraud** [comment]
Can you please make a pull request dedicated to the zoom buttons? (ie. with only the modifications for that specific issue)

**Pull request closed without merging**